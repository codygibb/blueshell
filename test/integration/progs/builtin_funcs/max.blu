print max(5, 6)
print max(8, 2)
print max(1.0, 1.1)
print max(1.0001, 1.0)
print max(3, 4.0)
