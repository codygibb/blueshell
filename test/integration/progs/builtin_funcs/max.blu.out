6
8
1.100000
1.000100
error@5: Incorrect_type (max, float, int)
